pkg/storaged/stratis/pool.jsx

```diff
@@ -64,6 +65,197 @@ const _ = cockpit.gettext;
 const fsys_min_size = 512 * 1024 * 1024;
 const fsys_round_size = 1024 * 1024;
 
+/* Abstractions over the r6 and r8 API revisions.
+ */
+
+function get_key_descriptions(pool) {
+    const result = [];
+
+    if (!pool.Encrypted)
+        return result;
+
+    if (client.stratis_interface_revision < 8) {
+        const val = pool.KeyDescription;
+        if (val[0] && val[1][0])
+            result.push({ slot: null, keydesc: val[1][1] });
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21927-ac5ce985-20250812-074807-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)